app/models/user.rb

```diff
@@ -72,8 +72,11 @@ class User < ApplicationRecord
   has_many :blog_posts, dependent: :destroy
 
   has_one :profile, dependent: :destroy
-  has_many :resumes, dependent: :destroy
+  has_many :resumes, dependent: :destroy, inverse_of: :user
   has_one :mentee_application, class_name: 'UserMenteeApplication', dependent: :destroy
+  # rubocop:disable Rails/InverseOf
+  has_one :current_resume, -> { where(current: true) }, class_name: 'Resume', dependent: nil
```

Comment:
shouldn't this by `find_by(current: true)` because it is a has_one?

Reply:
In general, you're correct, but when I used `find_by`, it throws up an exception. I think it's the way that the scope using a `has_one` functions. Even the documentation on `has_one` shows it that way:
https://api.rubyonrails.org/classes/ActiveRecord/Associations/ClassMethods.html#method-i-has_one-label-Scopes